server/src/main/scala/com/cloudera/livy/sessions/Session.scala

```diff
@@ -31,6 +31,7 @@ import org.apache.hadoop.fs.permission.FsPermission
 import org.apache.hadoop.security.UserGroupInformation
 
 import com.cloudera.livy.{LivyConf, Logging, Utils}
+import com.cloudera.livy.LivyConf._
```

Comment:
I'd just do `LivyConf.IMPERSONATION_ENABLED` since that's the only reference in this file...